CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Support for config options `[collectd]` and `[opentsdb]` has been removed; use `[[collectd]]` and `[[opentsdb]]` instead.
 * Config option `data-logging-enabled` within the `[data]` section, has been renamed to `trace-logging-enabled`, and defaults to `false`.
 * The keywords `IF`, `EXISTS`, and `NOT` where removed for this release.  This means you no longer need to specify `IF NOT EXISTS` for `DROP DATABASE` or `IF EXISTS` for `CREATE DATABASE`.
+* `max-series-per-database` was added with a default of 1M but can be disabled by setting it to `0`.
```

Comment:
As a side note, I made sure to test that. If you have over 1M series, you'll still be able to start up, you just won't be able to add new series.